lib/money/money.rb

```diff
@@ -168,7 +168,7 @@ def self.setup_defaults
     @use_i18n = true
 
     # Default to using legacy locale backend
```

Comment:
is `:currency` equivalent to `:legacy` here, or should this comment be removed?

Reply:
Yeah, let me create another PR to fix this.

Reply:
PR here: https://github.com/taskrabbit/money/pull/2